### PR TITLE
stack_depth takes a signed int as argument

### PR DESCRIPTION
**Pull request by tautschnig**

### CODE REVIEW

**Comment by kroening** [comment]
Should that one be unsigned?

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 09165fbb6350d40376600fae267229f51a3ef1f6).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/77134797

File: src/goto-instrument/stack_depth.h [hunk 1]
```diff
@@ -14,10 +14,12 @@ Date: November 2011
...
 #define CPROVER_GOTO_INSTRUMENT_STACK_DEPTH_H
 
+#include <cstdint>
+
 class goto_modelt;
 
 void stack_depth(
   goto_modelt &,
-  const int depth);
+  const std::size_t depth);
```
Comment (peterschrammel) [inline src/goto-instrument/stack_depth.h:23]:
```cl -DSATCHECK_MINISAT2  -DHAVE_MINISAT2 /W3 /D_CRT_SECURE_NO_WARNINGS /O2 /GF -I .. /nologo /c /EHsc stack_depth.cpp /Fostack_depth.obj
stack_depth.cpp
c:\codebuild\tmp\src717511119\src\github.com\diffblue\cbmc\src\goto-instrument\stack_depth.h(23): error C2039: 'size_t': is not a member of 'std'
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\cstdint(17): note: see declaration of 'std'
```

**Review by peterschrammel — APPROVED** [review]
LGTM modulo CI passing

**Pull request merged**